cmd/eno-reconciler/main.go

```diff
@@ -95,14 +96,14 @@ func run() error {
 		}
 	}
 
-	remoteConfig := mgr.GetConfig()
+	remoteConfig := rest.CopyConfig(mgr.GetConfig())
 	if remoteKubeconfigFile != "" {
 		if remoteConfig, err = k8s.GetRESTConfig(remoteKubeconfigFile); err != nil {
 			return err
 		}
-		if remoteQPS != 0 {
-			remoteConfig.QPS = float32(remoteQPS)
-		}
+	}
+	if remoteQPS >= 0 {
```

Comment:
The configured QPS should apply to the downstream client regardless of if its config was inferred from the environment or specified explicitly